spec/ddtrace/contrib/grape/tracer_spec.rb

```diff
@@ -196,6 +196,65 @@
     end
 
     context 'failure' do
+      context 'without filters' do
+        subject(:response) { post '/base/hard_failure' }
+
+        it 'should handle exceptions' do
+          expect(response.body).to eq('405 Not Allowed')
+          expect(spans.length).to eq(1)
+          expect(spans[0].name).to eq('grape.endpoint_run')
+          expect(spans[0].status).to eq(1)
+          expect(spans[0].get_tag('error.stack')).to_not be_nil
+          expect(spans[0].get_tag('error.type')).to_not be_nil
+          expect(spans[0].get_tag('error.msg')).to_not be_nil
```

Comment:
You can replace these with our [custom matchers](https://github.com/DataDog/dd-trace-rb/blob/master/spec/support/span_helpers.rb):

```suggestion
          expect(spans[0]).to_not have_error
          expect(spans[0]).to_not have_error_stack
          expect(spans[0]).to_not have_error_type
          expect(spans[0]).to_not have_error_message
```

We should also do it for the other tests below.